lib/leanplum_api/api.rb

```diff
@@ -176,20 +180,37 @@ def development_connection
     end
 
     # Deletes the user_id and device_id key/value pairs from the hash parameter.
-    def extract_user_id_or_device_id_hash!(hash)
-      user_id = hash.delete(:user_id) || hash.delete(:userId)
-      device_id = hash.delete(:device_id) || hash.delete(:deviceId)
-      fail "No device_id or user_id in hash #{hash}" unless user_id || device_id
+    # @param [Hash] user_hash
```

Comment:
Maybe something more informative than `user_hash`? `User attributes`, `User information`? `user_hash` looks more like a variable name than a description of an argument.

Reply:
changed